.docker/intelmq-full/Dockerfile

```diff
@@ -49,7 +48,7 @@ COPY ./intelmq-api /opt/intelmq-api
 
 RUN useradd -d /opt/intelmq -U -s /bin/bash intelmq \
     && adduser intelmq sudo \
-    && echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers \
+    && echo "intelmq ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/intelmq \
```

Comment:
this creates a security issue. intelmq must not run as root and should not run or required privileges

Reply:
I agree, but in this case intelmq is already in sudo group (line 50), in the previous version the entire grupo sudo could execute without password, now its more restricted, only to the intelmq user.
Probably we need to test and understand why it's in sudo.

Reply:
Okay, that's interesting.